.github/workflows/ci.yml

```diff
@@ -282,11 +310,57 @@ jobs:
           release_id: ${{ needs.draft_release.outputs.create_release_id }}
           file: target/${{ matrix.target }}/release/tailcall-${{ matrix.target }}${{ matrix.ext }}
           overwrite: true
+  release_lambda:
+    name: Release (Lambda)
+    needs: [test, draft_release, check_if_build, test_cf, build_lambda]
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main' && (needs.check_if_build.outputs.check_if_build == 'true')
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    env:
+      GITHUB_TOKEN: ${{secrets.GITHUBTOKEN}}
+      APP_VERSION: ${{ needs.draft_release.outputs.create_release_name }}
 
+    steps:
+      - name: Checkout Current Branch (Fast)
+        uses: actions/checkout@v4
+
+      - name: Install Correct Toolchain
+        uses: dtolnay/rust-toolchain@stable
```

Comment:
here is also need to replace action

Reply:
Fixed in 7747b20f9975ffa1dd077e08a559bfa27b0fb288